### DISCUSSION TITLE
Lesson 1 challenge

### DISCUSSION THREAD

**Discussion by SCPassion** [Cyfrin/foundry-full-course-cu#596] [Q&A] [answered]
I have some problem submitting transaction for lesson 1 challenge.
As far as I know, there is no particular task for lesson 1 challenge. I just have to type something as twitter handle.
But I am not able to submit the transaction. 

Could somebody provide me some guidelines?
![lesson1question](https://github.com/Cyfrin/foundry-full-course-f23/assets/141584825/021ed4fb-02be-4a24-8e0c-7ea6dae7f812)

**Comment by shikhar229169** [comment, marked as answer]
Actually you have already submitted the challenge earlier. See this transaction:
https://sepolia.etherscan.io/tx/0x4a529d4bdb0c10454628c36b369aaa453b8b5ece1876e6d1a07274c61547c585

You can't call solve challenge again if you have already done it before for a particular lesson.

**Reply by SCPassion** [reply]
Thanks for answering.

**Reply by dodeyeikoi** [reply]
after approving the transaction, where do i see the nft for the challenge please?

**Reply by SCPassion** [reply]
You could first go to your sepolia explorer. Then, trace back to the transaction that corresponding to the minting of challenge nft. Then, look for the contract address and the tokenId. Afterwards, go to your metamask wallet. Click the NFTs session, and select Import nft.
Paste your NFT contract address and tokenId. You will find your challenge nft. Congratulations and have fun. ^^
